web_src/less/_base.less

```diff
@@ -333,9 +333,9 @@ a.commit-statuses-trigger {
 }
 
 .issue-title code {
-  padding: 2px 4px;
-  border-radius: 6px;
-  background-color: var(--color-markup-code-block);
+  padding: 2px 4px !important;
+  border-radius: 6px !important;
+  background-color: var(--color-markup-code-block) !important;
```

Comment:
Well, personally, I do not like unclear `!important` styles.

Since the issue list on activity page are rendered by that page and they already have `code` style, I believe we do not need to touch the style again. 

Otherwise, on the activity page, there will be 2 different `code` styles.

If we really want to change the `code` style on the activity page, then the original style should be changed, but not adding these `!important`.

Reply:
# Current code style on main
![image](https://user-images.githubusercontent.com/20454870/195980121-9e791315-6e73-49a3-a03a-9ce30997ddc7.png)


# Current code style on this PR
![image](https://user-images.githubusercontent.com/20454870/195980089-16bdd87a-b018-4e08-b61b-fd0dbfced28f.png)

Which do you prefer?

Reply:
For me, adding `!important` to single-class selectors is totally fine.

Reply:
Do you want to have two different styled `code` on the feed list?

Reply:
I see, yes we should just remove these two conflicting styles:

```diff
diff --git a/web_src/less/_dashboard.less b/web_src/less/_dashboard.less
index 570d772729..fb5b4a8dd2 100644
--- a/web_src/less/_dashboard.less
+++ b/web_src/less/_dashboard.less
@@ -140,11 +140,9 @@
       font-family: var(--fonts-monospace);
     }

     code {
-      padding: 1px;
       font-size: 85%;
-      background-color: rgba(0, 0, 0, .04);
       border-radius: 3px;
       word-break: break-all;
     }

Reply:
Edited above, lets also remove `font-size`.

Reply:
I will make PR for this PR

Reply:
* https://github.com/yardenshoham/gitea/pull/4

![image](https://user-images.githubusercontent.com/2114189/195980660-e3ac4a79-901b-44eb-b291-52c5c8a2fdc7.png)

Reply:
I disagree about removing `!important`, it should stay as it adds resilience to the rule. Without `!imporant`, it's far to easy to break the style through other CSS interference. It definitely needs it. So I would only approve my diff above.

Reply:
It's unnecessary. Most time the `!important` used is improper. We have discussed many times. Like this one:
* https://github.com/go-gitea/gitea/pull/17866#issuecomment-983404274

And https://en.wikipedia.org/wiki/Occam's_razor  "entities should not be multiplied beyond necessity"


Every element could have their proper styles accurately.

Reply:
In most times, when `!important` is used, it means that there is some essential design problem in code.

Gitea's frontend is messy, do not make it more messy.

Reply:
Summarizing:
* All participants of this discussion agree the style needs to be changed. @silverwind says we should simply use `!important`, @wxiaoguang suggests yardenshoham/gitea#4.

I'm not a gitea maintainer (I'd prefer `!important`) so we need another maintainer's opinion to have a tie-breaker.

Reply:
I'm in favor of @wxiaoguang's solution.
For me, the less `!important`s, the better.
But, if the mentioned PR gets merged, you will have to adapt my suggestion above.